### ISSUE TITLE
Unexpected Abouts Screen pops-up upon changing the theme

### ISSUE THREAD

**Issue by shubhamnanche** [WirelessAlien/BhagavadGitaApp#16]
**Description of the bug**
When I try to change the theme of the app from `Light` to `Dark` and then again `Light`, the `About Gita` comes forth behind the alert dialog and the main activity disappears. Even when you click back, the app exists.

**To Reproduce**
1. Open the app
2. Go to the `About Gita` screen from the three-dot menu in the app bar
3. Go back to the Main Activity (Home Page)
4. Click on `Change Theme` options menu item in the app bar
5. Successively change the theme from Light -> Dark -> Light (This will trigger the `About Gita` page to pop)

**Expected behavior**
`About Gita` screen must not come forth and the app should remain in the Main Activity (i.e. Home screen)

**Comment by WirelessAlien** [comment]
Did you find this bug in MIUI device? I did some search about it maybe it is only an miui issue.

**Comment by WirelessAlien** [comment]
Did you find this bug in MIUI device? I did some search about it maybe it is only an miui issue.

**Comment by shubhamnanche** [comment]
Yes, I had tested it on MIUI. But, doesn't it have a solution though?
